### PR TITLE
Mttl 794

### PR DESCRIPTION
**Pull request by humulla**
Update parameter name to the new one

### CODE REVIEW

File: HousingSearchListener/V1/Interfaces/ElasticSearchHelper.cs [hunk 1]
```diff
@@ -13,7 +13,17 @@ public ElasticSearchHelper(IElasticClient elasticClient)
             _elasticClient = elasticClient;
         }
```
Comment (amidomichael) [inline HousingSearchListener/V1/Interfaces/ElasticSearchHelper.cs:15]: I don't think this class should live in the `Interfaces` folder

Comment (humulla) [reply HousingSearchListener/V1/Interfaces/ElasticSearchHelper.cs:15]: I've created a Helper folder and stored it there

File: HousingSearchListener/V1/Interfaces/ElasticSearchService.cs [hunk 2]
```diff
@@ -53,21 +53,33 @@ private async Task<HttpResponseMessage> GetPersonFromPersonApi(SQSEvent.SQSMessa
...
+            switch (personCreatedMessage.EventType)
+            {
+                case EventTypes.PersonCreatedEvent:
+                    await _esHelper.Create(esPerson);
+                    break;
+                case EventTypes.PersonUpdatedEvent:
+                    await _esHelper.Update(esPerson);
+                    break;
+                default:
+                    await _esHelper.Create(esPerson);
```
Comment (amidomichael) [inline HousingSearchListener/V1/Interfaces/ElasticSearchService.cs:74]: I think the default should be to do nothing...

Comment (evak6979) [reply HousingSearchListener/V1/Interfaces/ElasticSearchService.cs:74]: I would've set it to default, I am just worried about potentially not setting the eventtype on the message in one of the calling apps.

**Pull request merged**